students/17th/team1/seongyeonkang/user/views.py

```diff
@@ -0,0 +1,38 @@
+import json
+from json.decoder import JSONDecodeError
+
+from django.http        import JsonResponse
+from django.views       import View
+from django.db.utils    import DataError
+
+from user.models    import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data  = json.loads(request.body)
+        
+            if len(data['email']) > 30 or len(data['password']) > 18:
+                return JsonResponse({'MESSAGE' : 'DATA_TOO_LONG'}, status=400)
+                
+            if '@' not in data['email'] or '.' not in data['email']:
+                return JsonResponse({'MESSAGE' : 'INVALID_EMAIL'}, status=400)
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'MESSAGE' : 'EMAIL_ALREADY_EXIST'}, status=400)
```

Comment:
```suggestion
                return JsonResponse({'MESSAGE' : 'EMAIL_ALREADY_EXIST'}, status=409)
```
중복 문제니까 409로 해주시면 되겠네요

Reply:
아 코드 설정에 주의 깊지 못했습니다 감사합니다!